README.md

```diff
@@ -46,8 +48,19 @@ Say I’m a Hoodie contributor and find a typo somewhere. Instead of fixing the
 
 <p align="center"><img src="/assets/Issue-Done.png"></p>
 
+### Contributors
+
+Thank you to everyone who has helped with this project.
+<!-- Contributors START
+Michael_McCombie michaelmccombie https://twitter.com/michaelbuilds design
+Contributors END -->
+<!-- Contributors table START -->
+| <img src="https://avatars.githubusercontent.com/michaelmccombie?s=100" width="100" alt="Michael McCombie" /><br />[<sub>Michael McCombie</sub>](https://twitter.com/michaelbuilds)<br />🎨 |
```

Comment:
I think you are supposed to link the 🎨 to the contribution, in that case the logo? In that case I would use this URL: https://raw.githubusercontent.com/hoodiehq/first-timers-bot/51742c62ae3e4e2be7e58d170a9eab73a3871bf4/assets/avatar.png that will work, even if you rename or relocate the image in future, as it points to the specific commit it was added with